packages/component-library/stories/Button.stories.tsx

```diff
@@ -10,137 +10,67 @@ import * as React from "react"
 focusVisible // This causes the :focus-visible polyfill to load
 
 storiesOf("Button", module)
-  .add("Default", () => <Button label="Label" />, {
-    backgrounds: { disable: true },
-  })
-  .add("Full Width", () => <Button label="Label" fullWidth={true} />, {
-    backgrounds: { disable: true },
-  })
-  .add(
-    "Full Width + Icon",
-    () => <Button label="Label" fullWidth={true} icon={configureIcon} />,
-    {
-      backgrounds: { disable: true },
-    }
-  )
-  .add("Hyperlink", () => <Button label="Label" href="//example.com" />, {
-    backgrounds: { disable: true },
-  })
-  .add(
-    "Hyperlink w/ onClick",
-    () => (
+  .add("Default", () => <Button label="Label" />)
+  .add("Full Width", () => <Button label="Label" fullWidth={true} />)
```

Comment:
TODO for a follow-up: set the background by default on stories of reversed components

Reply:
Issue to resolve: https://github.com/cultureamp/kaizen-design-system/issues/81